src/librustc_error_codes/error_codes/E0718.md

```diff
@@ -1,7 +1,6 @@
-This error indicates that a `#[lang = ".."]` attribute was placed
-on the wrong type of item.
+A `#[lang = ".."]` attribute was placed on the wrong type of item.
```

Comment:
```suggestion
`#[lang = ".."]` attribute was placed on the wrong item type.
```

Reply:
Why not:

```suggestion
A `#[lang = ".."]` attribute was placed on the wrong item type.
```

?

Reply:
`A` feels weird, I don't think we need to quantify that. I mean from my reading point of view, I mean ideally we shouldn't read it aloud when we read (but see instead). Still, I find it shorter, faster and easier to understand without `A` for me.

Reply:
it reads fine to me tbh with `a` rather than without

Reply:
I applied the change I suggested then.